tests/phpunit/tests/media.php

```diff
@@ -5093,6 +5093,46 @@ static function( $res ) {
 		);
 	}
 
+	/**
+	 * @ticket 58635
+	 *
+	 * @covers ::wp_get_loading_optimization_attributes
+	 */
+	public function test_wp_get_loading_optimization_attributes_header_block_template_increase_media_count() {
+		$this->assertSame( 0, wp_increase_content_media_count( 0 ) );
+
```

Comment:
This can be removed, as the content media count gets reset before every test, via `set_up()`. No need to assert that here. That also improves the test to only have a single assertion.
```suggestion
```

Reply:
I just wanted to show value was increasing in the test. But you have a valid point, since it is in the `set_up()` it makes no sense to asset it. 

I have removed this line from both places.